src/ucp/api/ucp.h

```diff
@@ -1295,6 +1308,76 @@ ucs_status_t ucp_mem_unmap(ucp_context_h context, ucp_mem_h memh);
 
 /**
  * @ingroup UCP_MEM
+ * @brief list of UCP memory use advice.
+ *
```

Comment:
We don't allow only @brief description. You should include full description as well